UID2-3726 Add instructions on scraping metrics for Azure (#662)
IABTechLab/uid2docs: docs/guides/operator-guide-azure-enclave.md

> - For Azure virtual network deployment availability, check [Linux container groups](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-region-availability#linux-container-groups) in the Azure documentation. You can only deploy to regions with the **Confidential SKU** column set to **Y** in the table.

Author: @thomasm-ttd FYI I checked this page and I think it's updated, there is no "Confidential SKU" column/tables anymore. Should we consider removing this section? Do we still need to mention `Confidential SKU`?

Reviewer: The link has changed to : https://learn.microsoft.com/en-us/azure/container-instances/container-instances-resource-and-quota-limits#confidential-container-resources-preview
Change the text to: 
For Azure virtual network deployment availability, check [Linux container groups](https://learn.microsoft.com/en-us/azure/container-instances/container-instances-resource-and-quota-limits#confidential-container-resources-preview) in the Azure documentation to confirm the availability of Confidential Container regional support.

<AttestFailure />

### Scraping Metrics
The Private Operator for Azure exposes [Prometheus-formatted metrics](https://prometheus.io/docs/concepts/data_model/) on port 9080 through the `/metrics` endpoint. You can use a Prometheus-compatible scraper to collect and aggregate these metrics for your own needs. 

> This scraper will need to have access to the VNet that the operators are running in. It is not recommended to allow accessing the `/metrics` endpoint from the load balancer.

Reviewer 2: ```suggestion
The scraper must have access to the VNet that the Private Operator is running in. We do not recommend giving the load balancer access to the `/metrics` endpoint.
```

<AttestFailure />

### Scraping Metrics

> The Private Operator for Azure exposes [Prometheus-formatted metrics](https://prometheus.io/docs/concepts/data_model/) on port 9080 through the /metrics endpoint. You can use a Prometheus-compatible scraper to collect and aggregate these metrics for your own needs.

Reviewer: Suggest we say:
You can use a Prometheus-compatible scraper to collect and aggregate these metrics for your own needs. This scraper will need to have access to the VNet that the operators are running in. It is not recommended to allow access the /metrics endpoint from the load balancer.
